frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1000,6 +1033,7 @@ function ReaderFooter:textOptionTitles(option)
         frontlight_warmth = T(_("Warmth level (%1)"), symbol_prefix[symbol].frontlight_warmth),
         mem_usage = T(_("KOReader memory usage (%1)"), symbol_prefix[symbol].mem_usage),
         wifi_status = T(_("Wi-Fi status (%1)"), symbol_prefix[symbol].wifi_status),
+        page_turners_inverted = T(_("Page-turn buttons inverted (%1)"), symbol_prefix[symbol].page_turners_inverted),
```

Comment:
I wouldn't preemptively introduce naming discrepancies.

Reply:
okay, what do you mean? what discrepancy?

Reply:
`page_turners_inverted` vs `Page-turn buttons inverted`

Reply:
would you prefer `page_turners_inverted` be `page_turn_buttons_inverted` instead?

Reply:
Yes.